Markdown/Markdown.sublime-syntax

```diff
@@ -898,18 +915,285 @@ contexts:
+          \s*$          # any whitespace until EOL
+      captures:
+        2: markup.raw.code-fence.objc++.markdown-gfm punctuation.definition.raw.code-fence.begin.markdown
+        3: markup.raw.code-fence.objc++.markdown-gfm constant.other.language-name.markdown
+      embed: scope:source.objc++
+      embed_scope: markup.raw.code-fence.objc++.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: markup.raw.code-fence.objc++.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:c))
+          \s*$          # any whitespace until EOL
+      captures:
+        2: markup.raw.code-fence.c.markdown-gfm punctuation.definition.raw.code-fence.begin.markdown
+        3: markup.raw.code-fence.c.markdown-gfm constant.other.language-name.markdown
+      embed: scope:source.c
+      embed_scope: markup.raw.code-fence.c.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: markup.raw.code-fence.c.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:c\+\+|cpp))
+          \s*$          # any whitespace until EOL
+      captures:
+        2: markup.raw.code-fence.c++.markdown-gfm punctuation.definition.raw.code-fence.begin.markdown
+        3: markup.raw.code-fence.c++.markdown-gfm constant.other.language-name.markdown
+      embed: scope:source.c++
+      embed_scope: markup.raw.code-fence.c++.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: markup.raw.code-fence.c++.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:regexp|regex))
+          \s*$          # any whitespace until EOL
+      captures:
+        1: markup.raw.code-fence.regexp.markdown-gfm punctuation.definition.raw.code-fence.begin.markdown
+        2: markup.raw.code-fence.regexp.markdown-gfm constant.other.language-name.markdown
+      embed: scope:source.regexp
+      embed_scope: markup.raw.code-fence.regexp.markdown-gfm
+      escape: '^[ ]{,3}(\1)\s*$'
+      escape_captures:
+        0: markup.raw.code-fence.regexp.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
     - match: |-
          (?x)
           {{fenced_code_block_start}}
           ([\w-]*)     # any number of word characters or dashes
-          .*$          # all characters until eol
+          .*$          # all characters until EOL
       captures:
-        1: punctuation.definition.raw.code-fence.begin.markdown
-        2: constant.other.language-name.markdown
+        2: punctuation.definition.raw.code-fence.begin.markdown
+        3: constant.other.language-name.markdown
       push:
         - meta_scope: markup.raw.code-fence.markdown-gfm
```

Comment:
For the same reason, imo this should be `meta_content_scope`. And if it would otherwise match on the newline, the match probably needs to include `\n` also, but I'm not sure about that.